angular/src/app/service/api/cvstatus.service.ts

```
@@ -0,0 +1,16 @@
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CvstatusService extends BaseApiService {
+  changeUrl() {
+    return 'CvStatus'
+  }
+
+  constructor(http: HttpClient) {
+    super(http);
+  }
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)